tfjs-backend-webgpu/src/kernels/pool_webgpu_test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import * as tf from '@tensorflow/tfjs-core';
+import {test_util} from '@tensorflow/tfjs-core';
+import {describeWebGPU} from '../test_util';
+
+const expectArraysClose = test_util.expectArraysClose;
+
+describeWebGPU('pool', () => {
+  // For PoolWithFilterSizeEqualsOneProgram. This case will fail wasm, so keep
+  // it here. Wasm bug: https://github.com/tensorflow/tfjs/issues/5471.
```

Comment:
Is it possible to add this test generally and skip it on the wasm backend specifically?

Reply:
I think this is possible. How about I follow up this in another PR?

Reply:
OK, in that case perhaps leave a TODO here.

Reply:
Thanks, TODO added, and BUG is here: https://github.com/tensorflow/tfjs/issues/5506